packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -67,12 +72,20 @@ class ScaleUpdateDetails {
   /// greater than or equal to zero.
   final double scale;
 
+  /// The scale along the horizontal axis implied by the pointers in contact
```

Comment:
nit (here and below): first paragraph of a dart doc should be a one-line summary